### PR TITLE
Add pagination

### PR DESCRIPTION
**Pull request by mejackreed**
Note: This pr changes the default sort to latest_change which is different than requested by @blalbrit but works better with discovery dispatcher.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ gem 'stanford-mods'
 gem 'retries'
 gem 'druid-tools'
 gem 'whenever', :require => false
+gem 'kaminari'
```
Comment (drh-stanford) [inline Gemfile]: maybe add that we use it for pagination

File: spec/controllers/docs_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        expect(assigns(:deletes).count).to eq 1
+      end
+      it 'page' do
+        get :deletes, format: :json, per_page: 1, page: 2
+        expect(assigns(:deletes).first.druid).to eq 'druid:ff1111gg2222'
+        expect(assigns(:deletes).count).to eq 1
+      end
+    end
+  end
+end
```
Comment (drh-stanford) [inline spec/controllers/docs_controller_spec.rb]: is there a provision for too large a page size in the input parameters? is that the `max_paginates_per` setting?

File: spec/features/collections_controller_spec.rb [hunk 3]
```diff
@@ -22,7 +22,7 @@
     expect(response).to be_success
     results = JSON.parse(response.body)
     expect(results).to be_an(Array)
-    expect(results).to eq ['druid:bb1111cc2222', 'druid:dd1111ee2222']
+    expect(results).to eq ['druid:dd1111ee2222', 'druid:bb1111cc2222']
```
Comment (drh-stanford) [inline spec/features/collections_controller_spec.rb]: these are order-sensitive now, right? i'd add a comment to that effect

**Comment by drh-stanford** [comment]
Looks good to me 👍 . I added a few minor comments

File: spec/controllers/docs_controller_spec.rb (hunk 2 above)
Comment (mejackreed) [inline spec/controllers/docs_controller_spec.rb]: Yes, we max it out at 10,000

File: Gemfile (hunk 1 above)
Comment (mejackreed) [inline Gemfile]: added.

File: spec/features/collections_controller_spec.rb (hunk 3 above)
Comment (mejackreed) [inline spec/features/collections_controller_spec.rb]: Added

**Comment by mejackreed** [comment]
Thanks for the comments @drh-stanford . I've pushed up changes to address them.

**Pull request merged**